skimage/restoration/tests/test_restoration.py

```diff
@@ -87,6 +88,19 @@ def test_richardson_lucy():
     np.testing.assert_allclose(deconvolved, np.load(path), rtol=1e-3)
 
 
+def test_richardson_lucy_filtered():
+    test_img_astro = rgb2gray(astronaut())
+
+    psf = np.ones((5, 5)) / 25
+    data = convolve2d(test_img_astro, psf, 'same')
+    deconvolved = restoration.richardson_lucy(data, psf, 5,
+                                              filter_epsilon=1e-6)
+
+    path = image_fetcher.fetch('restoration/tests/astronaut_rl.npy')
```

Comment:
Here you can use the `fetch` function from `skimage._shared.testing` as in https://github.com/scikit-image/scikit-image/blob/master/skimage/data/tests/test_data.py#L144; the function skips the test if pooch is not installed. Almost there :-).

Reply:
@emmanuelle but do we want to skip the tests in CI? I'm kinda confused here as to why we *don't* have pooch as a test dependency, at least in *most* builds.

Reply:
It does not feel right that any test is skipped automatically if pooch is not installed -- especially if its implicit.  My preference would be to never skip and enforce pooch installation, or allow each to have a `skipif` explicitly inside so that people reading the code see the dependency.

In general, I'd be worried about long-run problems if the package depends on data downloaded outside of its own version control.

But I am happy to do whatever you feel is necessary at the moment, even if there are longer-term changes coming.  Just let me know the final decision.

Reply:
And to give a concrete suggestion, it might be worth replacing large test files with smaller ones and/or enforcing that future test files are below some threshold in size.  I chose to use the astronaut in this test because it was already being used for a test of similar functionality.  But there is no real reason I couldn't use a smaller image.

Reply:
@chrisroat I suggest just add pooch to the test requirements for now. We can revisit that decision in a future PR, since there's (as you can tell) a lot of tweaking to be done about pooch.